tests/MiniCover.ApprovalTests/_ApprovalTests/Reporters/TravisCIReporter.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using ApprovalTests.Core;
+
+namespace ApprovalTests.Reporters
+{
+    public class TravisCIReporter : IEnvironmentAwareReporter
+    {
+        public static readonly TravisCIReporter INSTANCE = new TravisCIReporter();
+
+        public void Report(string approved, string received)
+        {
+            ContinousDeliveryUtils.ReportOnServer(approved,received);
```

Comment:
I'm curious to understand what this line does.
Does it show results in any special way on TravisCI?

Reply:
It should for me but not sure it's working